components/brave_wallet/browser/keyring_service.cc

```diff
@@ -2599,6 +2599,14 @@ KeyringService::SignCip30MessageByCardanoKeyring(
   return result;
 }
 
+std::array<uint8_t, kPolkadotSubstrateAccountIdSize>
+KeyringService::GetPolkadotPubKey(const mojom::AccountIdPtr& account_id) {
+  CHECK(account_id);
+  auto* keyring = this->GetKeyring<PolkadotKeyring>(account_id->keyring_id);
```

Comment:
~this->~
Also please check keyring for nullptr

Reply:
Done!